doc/getting_started/up2.rst

```diff
@@ -43,159 +44,155 @@ the recommended options for choosing an Automatic installation type,
+   cma=1024M@0x30000000-0
 
-The UUID of the partition /dev/mmcblk0p3 can be found by
+The UUID of the partition ``/dev/mmcblk0p3`` can be found by
 
-.. code-block:: none
+.. code-block:: console
 
-      # blkid /dev/mmcblk0p3
+   # blkid /dev/mmcblk0p3
 
 
-Note that the Clear Linux system gets updated to the latest status when
-adding new packages with bundle-add. The SOS kernel version above is
-only an example. Look at the "/EFI/org.clearlinux/" under the EFI
+Note that the Clear Linux system gets updated to the latest version when
+adding new packages with ``bundle-add``. The SOS kernel version above is
+only an example. Look at the ``"/EFI/org.clearlinux/"`` under the EFI
 partition and adjust the kernel version.
 
-.. code-block:: none
-
-      # mount /dev/mmcblk0p1 /mnt
-      # ls /mnt/EFI/org.clearlinux
-      bootloaderx64.efi
-      kernel-org.clearlinux.native.4.15.9-538
-      kernel-org.clearlinux.pk414-sos.4.14.23-19
-      kernel-org.clearlinux.pk414-standard.4.14.23-19
-      loaderx64.efi
-
-Replace the ACRN devicemodel with the newly built one.
+.. code-block:: console
 
-.. code-block:: none
-
-      # cp build/devicemodel/acrn-dm /usr/bin/acrn-dm
+   # mount /dev/mmcblk0p1 /mnt
+   # ls /mnt/EFI/org.clearlinux
+   bootloaderx64.efi
+   kernel-org.clearlinux.native.4.15.9-538
+   kernel-org.clearlinux.pk414-sos.4.14.23-19
+   kernel-org.clearlinux.pk414-standard.4.14.23-19
+   loaderx64.efi
 
 Reboot UP2 board and select "The ACRN Service OS" to boot. From here you
 can login as root using the password you set previously when installing
 Clear Linux.
 
 Installing the UOS
-~~~~~~~~~~~~~~~~~~
+------------------
 
 Follow the ACRN getting started guide to `*create a bridge
 network* <https://projectacrn.github.io/latest/getting_started/#create-a-network-bridge>`__
 and `*setup a reference
 UOS* <https://projectacrn.github.io/latest/getting_started/#set-up-reference-uos>`__.
 
-Launching UOS in the GUI
-~~~~~~~~~~~~~~~~~~~~~~~~
+Launching UOS from the Service OS GUI
+-------------------------------------
 
 Mouse and keyboard have also been confirmed to work with the GUI
 control. Follow the `*Clear Linux
 guidance* <https://clearlinux.org/documentation/clear-linux/guides/maintenance/enable-user-space>`__
 on how to enable the GUI.
 
-.. code-block:: none
+.. code-block:: console
 
-      # useradd <userID>
-      # passwd <userID>
-      # usermod -G wheel -a <userID>
-      # swupd bundle-add desktop
-      # systemctl start gdm
+   # useradd <userID>
+   # passwd <userID>
+   # usermod -G wheel -a <userID>
+   # swupd bundle-add desktop
+   # systemctl start gdm
 
 When the GUI starts, login with the userID and password, launch a new
 terminal, and from there launch the UOS.
 
-.. code-block:: none
+.. code-block:: console
 
-      $ cd /usr/share/acrn/demo
-      $ sudo ./launch_UOS.sh
+   $ cd /usr/share/acrn/samples/nuc
+   $ sudo ./launch_UOS.sh
 
 |image0|
```

Comment:
Using a one-use replacement role is a contorted way to include an image.  Instead, write this as:

    .. image:: images/up2-gui.png
   
and delete the ``.. |image|`` directive below. I'd leave off the height/width arguments too.